### PR TITLE
Modify S878: Migrate to LayC - comma operator

### PR DESCRIPTION
**Pull request by frederic-tingaud-sonarsource**
## Review

A dedicated reviewer checked the rule description successfully for:

- [x] logical errors and incorrect information
- [x] information gaps and missing content
- [x] text style and tone
- [x] PR summary and labels follow [the guidelines](https://github.com/SonarSource/rspec/#to-modify-an-existing-rule)

### CODE REVIEW

**Review by yassin-kammoun-sonarsource — APPROVED** [review]
LGTM!

File: rules/S878/javascript/rule.adoc [hunk 1]
```diff
@@ -19,6 +28,11 @@ for(i = 0, j = 5; i < 6; i++, j++) { ... }
 
 * If the expression sequence is explicitly wrapped in parentheses.
 
+[source,javascript]
+----
+i = (a += 2, a + b); // Compliant by exception
+----
+
```
[the remark this answers is not in the document]
Comment (marco-antognini-sonarsource) [reply rules/S878/javascript/rule.adoc:35]: FYI According to the [doc](https://github.com/SonarSource/rspec/blob/master/docs/link_formatting.adoc#link-formatting), the correct format is:

> The hyperlink to anything apart from other Sonar rules should be applied to just the document name, with the 'source' being left as plain text. The idea is that this makes it really easy for a user to understand the source before they click on anything.

so "MDN -" should not be part of the clickable link.

File: rules/S878/javascript/rule.adoc [hunk 2]
```diff
@@ -19,6 +28,11 @@ for(i = 0, j = 5; i < 6; i++, j++) { ... }
 
 * If the expression sequence is explicitly wrapped in parentheses.
 
+[source,javascript]
```
[the remark this answers is not in the document]
Comment (marco-antognini-sonarsource) [reply rules/S878/javascript/rule.adoc:31]: Nicely spotted, the C example suffers from this too.

File: rules/S878/javascript/rule.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (yassin-kammoun-sonarsource) [reply rules/S878/javascript/rule.adoc:35]: Suggestion fixed. Thanks!

**Pull request merged**